travis/Dockerfile-travis-watch-build

```diff
@@ -21,5 +19,8 @@ RUN chown -R mitodl:mitodl /src
 USER mitodl
 
 COPY package.json yarn.lock ./webpack_if_prod.sh /src/
+COPY scripts /src/scripts
+
+RUN node /src/install_yarn.js
```

Comment:
this needs to be run as root, so can you copy the `USER mitodl` line below this?

the path to the script is going to be `/src/scripts/install_yarn.js`